presto-native-execution/presto_cpp/main/http/HttpServer.h

```diff
@@ -221,6 +221,16 @@ class DispatchingRequestHandlerFactory
       endpoints_;
 };
 
+class HttpConfig {
+ public:
+  HttpConfig(const folly::SocketAddress& httpsAddress);
+
+  proxygen::HTTPServer::IPConfig httpIpConfig() const;
```

Comment:
nit: since this is HttpConfig class so ipConfig() is good enough :)

Reply:
Make sense. Even for HttpsConfig class, I think we should rename to just `address` instead of `httpsAddress`. The reasoning is the class name already consists the name of protocol type.